### PR TITLE
build: Add Basic Dockerfile for NGINX

### PR DESCRIPTION
**Pull request by PCOffline**

### CODE REVIEW

**Review by carmeltsadok — CHANGES_REQUESTED** [review]
Add a copy of nginx configuration file.

**Review by carmeltsadok — CHANGES_REQUESTED** [review]
Add an nginx.conf file as well

File: nginx.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+server {
+  listen 3000;
+  root /usr/share/nginx/html;
+
+  location / {
+    index index.html index.html
```
Comment (carmeltsadok) [inline nginx.conf:6]:
```suggestion
    index index.html index.htm;
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM node:14-alpine as build
+WORKDIR /usr/src/app
+COPY ["package.json", "package-lock.json", "./"]
+ENV NODE_ENV=production
```
Comment (carmeltsadok) [inline Dockerfile:4]: It is not required when using tag prod

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (PCOffline) [reply Dockerfile:4]: As Yonatan said, it is to avoid installing devDependencies.

**Comment by PCOffline** [comment]
@carmeltsadok ready for merge with dev

**Review by carmeltsadok — APPROVED** [review]

**Pull request merged**